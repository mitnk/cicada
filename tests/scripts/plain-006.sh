echo test right head commands finishes first
yes | head -n 2
yes | head -n 2 | cat
yes | head -n 2 | cat | head | head
