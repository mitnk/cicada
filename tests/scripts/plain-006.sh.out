test right head commands finishes first
y
y
y
y
y
y
